source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -601,6 +627,8 @@ class Filter : public Logger::Loggable<Logger::Id::ext_proc>,
   std::vector<std::string> untyped_forwarding_namespaces_{};
   std::vector<std::string> typed_forwarding_namespaces_{};
   std::vector<std::string> untyped_receiving_namespaces_{};
+  std::vector<std::string> untyped_cluster_metadata_forwarding_namespaces_{};
```

Comment:
I'd recommend using a struct with two fields since it's too easy to confused these two fields somewhere.

Reply:
good point, let me do it in another PR tho. 
in this PR let's follow the same pattern. 
I have https://github.com/envoyproxy/envoy/issues/41988 on my table, I can do the cleanup as suggested in that issue.